### PR TITLE
Show full Repo Description

### PR DESCRIPTION
**Pull request by skyGtm**
Repository description is usually unreadable, because the sentence is incomplete. so this commit will show repo description upto 10 lines, even if the description exceeds more than 10 lines, the sentence will be more readable than before

### CODE REVIEW

**Comment by Iamlooker** [comment]
I will have to check how this looks

File: app/src/main/res/layout/repository_item.xml [hunk 1]
```diff
@@ -22,7 +22,8 @@
         android:id="@+id/repository_description"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:singleLine="true"
+        android:singleLine="false"
+        android:maxLines="10"
```
Comment (Iamlooker) [inline app/src/main/res/layout/repository_item.xml:26]: max lines of 2 looks very consistent and decent, I would rather prefer that.

File: app/src/main/res/layout/repository_item.xml [hunk 2]
```diff
@@ -22,7 +22,8 @@
...
         app:layout_constraintBottom_toBottomOf="parent"
```
Comment (Iamlooker) [inline app/src/main/res/layout/repository_item.xml:27]: Also I would like if you could add an ellipsize at the end

**Review by Iamlooker — CHANGES_REQUESTED** [review]
I will merge as soon as the requested points are matched :)

**Comment by skyGtm** [comment]
Changes: default value of singleLine property is false so it is removed.

with maxLines 2 and ellipsize at end, text is readable but still incomplete.
Using collapsable view will be good solution if the user ever wanted to read full description.

## Full Screenshot (Click Here)
<details><summary>CLICK ME</summary>
<p>
full_shot
</p>
</details>

**Comment by Iamlooker** [comment]
Actually you can long click to the repository to see more details

**Comment by Iamlooker** [comment]
I am planning to change some ui

**Pull request merged**